### PR TITLE
Feat/license unlicense

### PR DESCRIPTION
**Pull request by streamich**

### CODE REVIEW

File: packages/mrm-task-license/index.spec.js [hunk 1]
```diff
@@ -10,11 +10,18 @@ const path = require('path');
 const { getConfigGetter } = require('mrm');
 const vol = require('memfs').vol;
 const task = require('./index');
+const { json: jsonFile } = require('mrm-core');
```
Comment (sapegin) [inline packages/mrm-task-license/index.spec.js]: I think it’s better to rename `json` variable but use Mrm methods consistently.

**Review by sapegin — CHANGES_REQUESTED** [review]
Thanks for the PR! 

One small comment, and please remove changes in lock files.

**Comment by streamich** [comment]
@sapegin done

**Review by sapegin — APPROVED** [review]

**Comment by sapegin** [comment]
Thansk, merged!

**Pull request merged**